### PR TITLE
Update Cocos2dxWebView.java

### PR DESCRIPTION
**Pull request by zchajax**
Before modification，When WebView displays, click the return key, and the game will exit directly. After modification, Clicking the return key calls the call back event listener function in the game, So you can click to return to the the previous page. e.g：

```
if (keycode == EventKeyboard::KeyCode::KEY_BACKSPACE) 
    {    
        if (m_WebView)  // if the webView was added                      
        {
            if (m_WebView->canGoBack()) // Determine whether to return to the previous level page
            {
                m_WebView->goBack();
            }
            else
            {
                m_WebView->runAction(RemoveSelf::create());    // close the webView

                m_WebView = nullptr;
            }
        }
        else 
        {
            Director::getInstance()->popScene(); // Quit the game or do something else
        }  
    }
```

### CODE REVIEW

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java [hunk 1]
```diff
@@ -67,6 +67,22 @@ public Cocos2dxWebView(Context context, int viewTag) {
 
         this.setWebViewClient(new Cocos2dxWebViewClient());
         this.setWebChromeClient(new WebChromeClient());
+        
+        this.setOnKeyListener(new OnKeyListener() 
+        {
+            @Override
+            public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event)
+            {
+                if (keyCode == KeyEvent.KEYCODE_BACK) // 返回键
```
Comment (Dimon4eg) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: Please write comments in English

**Review by Dimon4eg — CHANGES_REQUESTED** [review]
Please write comments in English

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java (hunk 1 above)
Comment (minggo) [reply cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: Good catch. Yep, should write comment in English.

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java [hunk 2]
```diff
@@ -67,6 +67,23 @@ public Cocos2dxWebView(Context context, int viewTag) {
...
+        
+        // handle the key event
+        this.setOnKeyListener(new OnKeyListener() 
+        {
+            @Override
+            public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event)
+            {
+                if (keyCode == KeyEvent.KEYCODE_BACK)
+                {
+                    Cocos2dxGLSurfaceView.getInstance().onKeyDown(keyCode, KeyEvent)
```
Comment (minggo) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: lost `;`

File: cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java [hunk 3]
```diff
@@ -67,6 +67,23 @@ public Cocos2dxWebView(Context context, int viewTag) {
...
+
+                    return true
```
Comment (minggo) [inline cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java]: lost `;`

**Comment by minggo** [comment]
Have you tested the codes? It has obvious compiling errors.

**Comment by minggo** [comment]
```
[javac] /home/travis/build/cocos2d/cocos2d-x/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:75: error: cannot find symbol
    [javac]             public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event)
    [javac]                                  ^
    [javac]   symbol: class DialogInterface
    [javac] /home/travis/build/cocos2d/cocos2d-x/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:75: error: cannot find symbol
    [javac]             public boolean onKey(DialogInterface dialog, int keyCode, KeyEvent event)
    [javac]                                                                       ^
    [javac]   symbol: class KeyEvent
    [javac] /home/travis/build/cocos2d/cocos2d-x/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:77: error: cannot find symbol
    [javac]                 if (keyCode == KeyEvent.KEYCODE_BACK)
    [javac]                                ^
    [javac]   symbol: variable KeyEvent
    [javac] /home/travis/build/cocos2d/cocos2d-x/cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxWebView.java:79: error: cannot find symbol
    [javac]                     Cocos2dxGLSurfaceView.getInstance().onKeyDown(keyCode, KeyEvent);
    [javac]                                                                            ^
    [javac]   symbol: variable KeyEvent
    [javac] Note: Some input files use unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 4 errors
    [javac] 3 warnings
```

I will close this PR since it is not tested. Always has obvious compiling errors.

**Pull request closed without merging**